web/source/kmwembedded.js

```diff
@@ -32,39 +32,39 @@
     if(device.app.indexOf('Mobile') >= 0) device.formFactor='phone';
     if(device.app.indexOf('Tablet') >= 0) device.formFactor='tablet';
     device.browser='native';
-  }
+  };
```

Comment:
These `;` additions are good tidyup but really belong in their own PR. Happy to leave them for now, but I think code cleanup like this should almost always be in its own PR in future.